dotnet/src/Functions/Functions.UnitTests/OpenApi/OpenApiDocumentParserV30Tests.cs

```diff
@@ -500,6 +501,86 @@ public async Task ItCanParseDocumentWithMultipleServersAsync()
         Assert.Equal("https://ppe.my-key-vault.vault.azure.net", restApi.Operations[0].Servers[1].Url);
     }
 
+    [Fact]
+    public async Task ItCanParsePathItemPathParametersAsync()
```

Comment:
- Please test that override works:
![image](https://github.com/user-attachments/assets/c7b95cf7-99a0-4e66-b8c2-928450ea7b64)
- Add similar tests to OpenApiDocumentParserV20Tests and OpenApiDocumentParserV31Tests tests classes to ensure that fix works for those versions as well.
- nit: Consider adding a new path to the documentV2_0.json, documentV3_0.json, and documentV3_1.yaml test documents and use them instead of declaring the document in the tests. This will make the tests a little tidier and may uncover some issues when the documents are loaded by the other tests.

Reply:
added the additional tests.
Did not implement a common resource since the value is probably to parse different document versions anyway.
Also having documents inline makes reading the test easier (everything is in context) and once you have read the document, you can collapse the string with the IDE.